frontend/src/components/activity/content/Notes.vue

```diff
@@ -1,26 +1,26 @@
 <template>
-  <card-content-node v-bind="$props">
+  <ContentNodeCard v-bind="$props">
     <api-richtext
       :uri="contentNode._meta.self"
       fieldname="data.html"
       :label="$tc('contentNode.notes.name')"
       rows="4"
       auto-grow
       :disabled="layoutMode || disabled"
-      :filled="layoutMode"
+      :readonly="layoutMode"
```

Comment:
Now the api-richtext is both disabled and readonly. Is this what we expect it to do? Wouldn't either disabled or readonly be sufficient?

Reply:
Should I solve this in this PR?